src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
-body {
+* {
```

Comment:
Consider using a more specific selector than '*' to avoid unintended styling of all elements. This is a general best practice to prevent potential conflicts.